### DISCUSSION TITLE
ScriptReference/KeyCode.Tab

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#600] [Notes]
# ScriptReference/KeyCode.Tab

https://docs.unity3d.com/ScriptReference/KeyCode.Tab.html

**Comment by FreyaHolmer** [comment]
If you want to override the tab key in custom editor scripts, `Event.current.keyCode == KeyCode.Tab` probably won't work the way you expect, likely because unity uses it to change focus of controls

In my case, I solved it by using `Event.current.character == '\t'` instead
